src/commands/ban.ts

```diff
@@ -24,20 +25,25 @@ export default {
         if (user.id === message.author.id) {
             return message.channel.send("Aww, please don't ban yourself! 💖");
         }
-        /* This checks if the user to be banned
-        is in the guild, and if true,
-        it checks if they can be banned by the bot or not. */
-        if (!message.guild.member(user)?.bannable) {
-            return message.channel.send(
-                '❌ I cannot ban this user! \n**Please make sure that my highest role is above theirs.**',
-            );
-        }
+
+        /* Attempts to ban the user, if the ban is successful,
+        the bot will send a message indicating it was successful, otherwise an error message will be sent
+         */
```

Comment:
Honestly, I would much rather see the closing `*/` in the line above, instead of having it be on its own separate line.